WordPress/Classes/System/SplitViewRootPresenter.swift

```diff
@@ -19,6 +19,13 @@ final class SplitViewRootPresenter: RootViewPresenter {
         let tabBarVC = WPTabBarController(staticScreens: false)
         splitVC.setViewController(tabBarVC, for: .compact)
 
+        NotificationCenter.default.publisher(for: MySiteViewController.didPickSiteNotification).sink { [weak self] in
```

Comment:
I resorted to using notifications for now. It's not ideal, but I think it should be OK for this one use-case. In the future PRs, we should add some global class for managing site switching and creation. Its' currently hardcoded in `SiteSwitcherViewController` and I didn't want to change it.